packages/ui-markdown-editor/src/utilities/constants.js

```diff
@@ -4,6 +4,8 @@ const MISC_CONSTANTS = {
   DROPDOWN_NORMAL: 'Normal',
 };
 
+export const SPACE_BAR = " ";
```

Comment:
Slightly nit-picky, but maybe this variable could be `SPACE_CHARACTER` instead.

Reply:
I changed it to a `SPACE_CHARACTER`. 👍